src/run/src/plugin.ts

```diff
@@ -86,7 +86,11 @@ export const plugin = async (
   }
 }
 
-const add = async (args: string[], config: LoadedConfig, quiet = false) => {
+const add = async (
+  args: string[],
+  config: LoadedConfig,
+  quiet = false,
+) => {
```

Comment:
This change is unrelated, it's simply the outcome of running `npm run format` as requested in the contribution instructions.